src/components/Home/Home.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import Home from './Home';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
Great job team🤝

- [ ] As stated in the project requirements, kindly use Jest snapshots to test the components.